application/system_processor/Cdl.py

```
@@ -0,0 +1,90 @@
+        self.goToPosition(position)
+    
+    def stepOverBackward(self, position):
+        '''
+            Step over function calls backwards
+        '''
+        originalStackSize = len(self.decoder.callStacks[position])
+
+        while position > 0:
+            position = self.decoder.getPreviousExecutionPosition(position)
+
+            if position == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)